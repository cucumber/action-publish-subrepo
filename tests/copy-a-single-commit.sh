#!/bin/bash

cd monorepo || exit
mkdir ruby
echo "# Ruby project" >ruby/README.md
git add .
git commit --message "Add ruby"

mkdir php
echo "# PHP project" >php/README.md
git add .
git commit --message "Add php"

# run the tool
publish-to-subrepo php "$SUBREPO"
