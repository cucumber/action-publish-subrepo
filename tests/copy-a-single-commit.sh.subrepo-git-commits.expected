
**
Add php



diff --git a/README.md b/README.md
new file mode 100644
--- /dev/null
+++ b/README.md
@@ -0,0 +1 @@
+# PHP project
